src/scripts/main.js

```diff
@@ -1 +1,69 @@
 'use strict';
+
+const logo = document.querySelector('.logo');
+
+function showMessage(className, innerText, notificationMessage) {
+  const message = document.createElement('div');
+
+  message.className = className;
+  message.innerText = innerText;
+  message.dataset.qa = 'notificationMessage';
```

Comment:
Notify users about resolved or rejected promises using DOM. For messages add QA attributes data-qa="notification"